README.md

```diff
@@ -47,7 +47,7 @@ a variety of nGraph-enabled backends: CPU, GPU, and custom silicon like the
 This will produce something like this:
 
         TensorFlow version:  1.13.1
-        nGraph bridge version: b'0.12.0-rc6'
+        nGraph bridge version: b'0.12.0'
```

Comment:
Right now after `CXX11_ABI` prints we also get a new line: `nGraph bridge built with Grappler: True`
Should we update the README?

Reply:
We will have t redo the README ... for now we will leave it as is